plugins/mathjax/plugin.js

```diff
@@ -306,55 +307,85 @@
+                                    var MathJax = {
+                                options: {
+                                    enableMenu: false,
+                                    ignoreHtmlClass: 'tex2jax_ignore',
+                                    processHtmlClass: 'tex2jax_process'
+                                },
+                                startup: {
+                                    pageReady: function () {
+                                        MathJax.startup.promise.then(function () {
+                                            getCKE().tools.callFunction(${loadedHandler});
+                                        },
+                                        function (err) {
+                                            console.log('Typeset failed: ' + err.message)
+                                        }
+                                      )
+                                    }
+                                }
+                              };
+                              function update() {
+                              MathJax.startup.promise.then(function () {
+                                var buffer = this.buffer;
+                                if (!Array.isArray(buffer)) buffer = [buffer];
+                                MathJax.typesetPromise(buffer).then(
+                                    function () {
+                                        getCKE().tools.callFunction(${updateDoneHandler});
+                                    },
+                                    function (err) {
+                                        console.log('Typeset failed: ' + err.message)
+                                        console.log(this.buffer)
+                                        console.log(typeof this.buffer)
+                                    }
+                                );
+                              },
+                              function (err) {
+                                  console.log('Typeset failed: ' + err.message);
+                              }
+                              );
+                              }
+                            `
+							: `
+                            MathJax.Hub.Config( {
+                                showMathMenu: false,
+                                messageStyle: "none"
+                            }
+                            );
+                            function update() {
+                                MathJax.Hub.Queue([ 'Typeset', MathJax.Hub, this.buffer ],
+                                function() {
+                                    getCKE().tools.callFunction(${updateDoneHandler});
+                                }
+                                );
+                            }
+
+                            MathJax.Hub.Queue( function() {
+                                    getCKE().tools.callFunction(${loadedHandler});
+                            });
+                            `
+					}
```

Comment:
As template strings can't be used, this fragment would become unreadable. I'd suggest moving it to a separate variable:

```javascript
var mathJaxInitCode = '[code for v2]';

if ( editor.config.mathJaxVersion === 3 ) {
	mathJaxInitCode = '[code for v3]';
}

doc.write( '[…]' );
```